docs/tutorials.rst

```diff
@@ -7,12 +7,52 @@ Tutorials
 
    notebooks/quickstart
    notebooks/governance_integration
-   notebooks/lens_customization
    notebooks/integration_demo
+   notebooks/lens_customization
 
 
 .. toctree::
    :caption: Demo Use Cases
    :maxdepth: 1
 
    notebooks/lens_demos/binaryclassification
+   docs/notebooks/module_demos/fairness_nlp
```

Comment:
this should be `notebooks/module_demos...` not `docs/notebooks/module_demos...`

However, I don't think we should put `module_demos` here. We should have another section for them. Using modules directly should be hidden as much as possible (that's the current working hypothesis anyway). I want to direct people to use lens.

Reply:
Agreed. Need to please both people who are more into exploring repos and others who prefer readthedocs.

Reply:
link fixed